### PR TITLE
feat: bubble android fullscreen events outside component

### PR DESCRIPTION
**Pull request by dickie81**
Exposes fullscreen events outside component for Android only.

### CODE REVIEW

File: packages/brightcove-video/brightcove-player.android.js [hunk 1]
```diff
@@ -58,7 +67,13 @@ function withNativeCommand(WrappedComponent) {
 
       const props = Object.assign(
         { runNativeCommand: this.runNativeCommand },
-        this.props,
+        Object.keys(this.props) // filter out android only props
+          .filter(key => Object.keys(propTypes).includes(key))
```
Comment (craigbilner) [inline packages/brightcove-video/brightcove-player.android.js]: which test covers this change?

File: packages/brightcove-video/__tests__/brightcove-player.android.test.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+    const rootInstance = brightcoveVideo.getInstance();
+
+    rootInstance.runNativeCommand("play", []);
+
+    expect(mockNativeFunc.mock.calls.length).toEqual(1);
+    expect(mockNativeFunc.mock.calls[0][1]).toEqual("PLAY_COMMAND");
+  });
+
+  it("will go fullscreen if it recieves an 'onChange' event with 'isFullscreen' equal to true", () => {
```
Comment (craigbilner) [inline packages/brightcove-video/__tests__/brightcove-player.android.test.js]: `receives` (for each test)

File: packages/brightcove-video/__tests__/brightcove-player.android.test.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    );
+
+    const rootInstance = brightcoveVideo.getInstance();
+
+    rootInstance.onChange({ isFullscreen: true });
+
+    expect(brightcoveVideo.toJSON()).toMatchSnapshot();
+  });
+
+  it("will fire event if it recieves an 'onChange' event with 'isFullscreen' equal to true", done => {
```
Comment (craigbilner) [inline packages/brightcove-video/__tests__/brightcove-player.android.test.js]: do these tests do what they say on the tin?

**Comment by dickie81** [comment]
@craigbilner Added extra test to ensure props correctly filtered.

**Pull request merged**